src/components/layout/NavBar.less

```diff
@@ -0,0 +1,58 @@
+.nav-bar {
+  width: 100%;
+  background-color: white;
+  border-bottom: 1px solid #d1d1d1;
+}
+
+.nav-bar__nav--left {
+  float: left;
+  display: inline-flex;
+}
+
+.nav-bar__nav--right {
+  float: right;
+  color: white;
+  overflow: hidden;
+}
+
+.nav-bar__header {
+  overflow: hidden;
+  margin: auto;
+  width: 1220px;
+}
+
+.nav-bar__link {
+  color: #606060;
+  display: inline-block;
+  text-align: center;
+}
+
+.nav-bar__nav--right .nav-bar__link {
+  border-left: 1px solid #d1d1d1;
+  &:last-child {
```

Comment:
can we unnest 😬

Reply:
Thanks! I will unnest `.nav-bar__nav--right .nav-bar__link`. 
But I think it is reasonable to nest pseudo class names like `:hover` or `:active` because they only effect element itself and won't cause any side effects. Can we keep them?

Reply:
yeah I get that, but it doesn't don't adhere to the BEM rules, and also doesn't give us the preprocessor speed.

Reply:
I think you are right about preprocessor speed, That's a good reason! 
Also if we want to have csslint in future then we should clean all `&` first (will be in next pr)
But I think BEM rule still allow reasonable nesting cases like this one: http://getbem.com/faq/#block-modifier-affects-elements